pkg/k8sclient/k8sclient.go

```diff
@@ -32,14 +55,22 @@ func GetNodeByLabel(label string, kubeClient kubernetes.Interface) (v1.Node, err
 	case 1:
 		return nodeList.Items[0], nil
 	default:
-		for _, node := range nodeList.Items {
-			nodeCapacity, ok := node.Status.Capacity[LvCapacity]
-			if !ok {
-				continue
+		if selectorMethod == RR {
+			if roundRobinIterator >= len(nodeList.Items){
```

Comment:
I worry about thread safety

would it be somehow possible to do quasi round robin without using a non-protected, package wide global variable?

Reply:
like: https://github.com/hlts2/round-robin

Reply:
or maybe this one: https://github.com/sbabiv/roundrobin
it uses mutexes internally

store a reference to an RR object inside the client, rather than making it a global variable